GettingStarted.md

```diff
@@ -32,31 +35,33 @@ If you have not already, download and install [Visual Studio 2017](https://www.v
 
 You can install more components and UWP SDK's as you wish.
 
-Make sure you are running the appropriate version of Unity 3D on your machine. You should [download and install the latest version](https://unity3d.com/get-unity/download/archive) this project says it supports on the [main readme page](/README.md).
+## 2.3 Install appropriate version of Unity 3D
 
 [unity-release]:             https://unity3d.com/unity/qa/patch-releases/2017.2.1p2
```

Comment:
These should change to 2017.4 LTS

Reply:
Fixed for this file only. I suggest we move the whole github project forward to 2017.4 with a separate PR (tied to a github issue). That would be a good time to enforce version consistency with a simple CI test (ensuring version matches between markdown documentations and ProjectVersion.txt) .